nucleus/payara-modules/notification-core/pom.xml

```diff
@@ -71,7 +71,14 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>18.0</version>
+            <version>19.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.2.6</version>
```

Comment:
that's the same case as guava version. we need to decide on where that should land in project-wise.